packages/core/upload/server/services/upload.js

```diff
@@ -21,6 +21,7 @@ const {
   errors: { ApplicationError, NotFoundError },
   file: { bytesToKbytes },
 } = require('@strapi/utils');
+const stream = require('stream');
```

Comment:
also should probably just destructure { Readable } from stream for tree shakeability.